packages/models-library/src/models_library/services_db.py

```diff
@@ -34,6 +35,7 @@ class Config:
                 "thumbnail": "http://thumbnailit.org/image",
                 "created": "2021-01-18 12:46:57.7315",
                 "modified": "2021-01-19 12:45:00",
+                "deprecated": datetime.fromisoformat("2099-01-19 12:45:00"),
```

Comment:
not sure why you need to use datetime here... the example is usually directly a dict

Reply:
will change